packages/api/lambdas/sf-scheduler.js

```diff
@@ -1,23 +1,28 @@
 'use strict';
 
 const get = require('lodash/get');
+const isNil = require('lodash/isNil');
 
+const { getCollection } = require('@cumulus/api-client/collections');
 const SQS = require('@cumulus/aws-client/SQS');
 const { buildQueueMessageFromTemplate } = require('@cumulus/message/Build');
-const isNil = require('lodash/isNil');
-const Collection = require('../models/collections');
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/api/lambdas/sf-scheduler' });
 const Provider = require('../models/providers');
 
 const getProvider = (id) => {
   if (isNil(id)) return undefined;
   return (new Provider()).get({ id });
 };
 
-const getCollection = (collection) => {
+const getApiCollection = (collection) => {
   if (isNil(collection)) return undefined;
-
-  const c = new Collection();
-  return c.get({ name: collection.name, version: collection.version });
+  return getCollection({
```

Comment:
Man, this is really excellent work catching all of this stuff. It also seems like we need to update the `getProvider` call in here, which is definitely out of scope for this PR but was missed by me. I can submit a small amendment PR

Reply:
👍🏻  thanks - and I think I can just throw up the PR, I already have the fix stashed.   I'll comment here when that's done.